components/template/Templates.vue

```diff
@@ -1,47 +1,63 @@
 <template>
   <div>
-    <Hero @searchInput="handleSearch" />
-    <div class="template_container d-flex">
+    <div class="container template-container d-flex">
       <div class="left-sidebar">
         <TemplateCategories
           :activeCategory="activeCategory"
           :templateCategories="templateCategories"
         />
       </div>
-      <div class="w-100">
-        <h2 class="heading">
-          {{
-            $route.params.slug ? $route.params.slug.replace('-', ' ') : 'All'
-          }}
+      <div class="content-wrapper w-100">
+        <div
+          v-if="activeCategory"
+          class="breadcrumb d-flex align-items-center gap-2"
+        >
+          <NuxtLink to="/templates/" class="breadcrumb-text">
+            All Templates
+          </NuxtLink>
+          <img src="~/assets/images/icons/chevron-right.svg" />
+          <span class="breadcrumb-text">{{ activeCategory.name }}</span>
+        </div>
+        <h1 class="content-heading mt-2">
+          {{ activeCategory ? activeCategory.name : 'All' }}
           Templates
-        </h2>
-        <div v-if="filterTemplates && filterTemplates.length" class="templates">
+        </h1>
+        <!-- <p class="content-description">
+          {{ activeCategory.description }}
+        </p> -->
```

Comment:
@dhaladitya108 seems like we are rendering the full description without the capability built for `see more` as per the design?

Reply:
yes, we do not have the see more option now. We have to add it.